packages/nuejs.org/docs/template-syntax.md

```diff
@@ -335,7 +335,7 @@ Then the final rendered button would merge both classes:
 
 
 ### Passing data with `:bind` { #bind }
-Bind directive makes every object property directly accessible on the component. Instead of writing `{ data.title }` inside the component, you can just write `{ title }`. This is particularly useful when looping components.
+Bind directive makes every object property directly accessible to the component. Instead of writing `{ data.title }` inside the component, you can just write `{ title }`. This is particularly useful when looping components.
```

Comment:
```suggestion
The bind directive makes every object property directly accessible from the component. Instead of writing `{ data.title }` inside the component, you can just write `{ title }`. This is particularly useful when looping components.
```

Hm. I think we used different points of view. You from the parent component and me from the child component?